# SPAAM2

## Programme

### Session Structure

**Provisional - to be confirmed**

* _09:00_ Introduction
* _09:15_ x2-3 12 minute talks + 3 minutes specific questions (45m)
  * Challenges encountered in own work
  * Attempted solutions (success or failure)
* _10:00_ Break
* _10:15_ Round table discussion (2 h)
  * Topics to address gathered prior to event
* _12:15_ Lunch (see [here](spaam2/foodandactivities.md) for more information)
* _13:45_ x1 12 minute talk _ 3 minute specific questions
* _14:00_ Round table discussion (2h)
* _16:00_ 
* _17:00_ Social activity (to be confirmed)

### Outline

**Provisional - to be confirmed**

Length: 2 days

Four sessions: 
* Biases in ancient metagenomics research
  * Pros and cons of different taxonomic classifiers
  * How to detect and remove contamination within databases
  * How to account for missing reference genomes
* Authentication of ancient metagenomic datasets
  * Define standard set of authentication criteria (metagenomics/single genomes)
  * Define standard control datasets for analysis and software testing
  * Non-bacterial organisms?
* #openscience
  * Reproducibility
  * Standards in reporting
* Building networks
  * Internally in field (ISCB/COSI?)
  * Bridging aDNA community and modern fields


